org.eclipse.scout.rt.api/src/main/java/org/eclipse/scout/rt/api/meta/IApiVersion.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2010, 2024 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.scout.rt.api.meta;
+
+import org.eclipse.scout.rt.platform.ApplicationScoped;
+
+/**
+ * Defines the version of the API. Can be used to check compatibility.
+ * <p>
+ * Replace this bean to change the version in project code.
```

Comment:
"Implement this bean ro provide a custom API version"  or "replace scout default implementation to provide a custom version"